Makefile

```diff
@@ -112,6 +112,11 @@ push-image:
 
 .PHONY: html-email
 html-email:
+	go run ./scripts/generatemjml/main.go \
+		-i resources/authgear/mjml/templates \
+		-t resources/authgear/mjml/translations \
+		-o resources/authgear
+
```

Comment:
- Store .mjml.gotemplate next to .mjml.
- Store translation.json next to .mjml.gotemplate

Reply:
You mean prefer keeping the generated `.mjml` in repo and placing `.mjml.gotemplate` next to them?

Reply:
Yes.

Reply:
I mean the following structure

```
/authgear
  /templates
    /en
      /messages
         forgot_password_sms.txt
         translation.json
```